requirements.txt

```diff
@@ -1,4 +1,5 @@
 sklearn
 fastapi[all]
 pandas
-numpy
+pycricbuzz
+scrapy
```

Comment:
Probably better to specify versions here as well

Reply:
Right
Thought so too